### ISSUE TITLE
0.112.0 - Console errors, no UI menu, and sidebar stays

### ISSUE THREAD

**Issue by iderdik** [maykar/custom-header#231]
Home Assistant 0.112.0.dev20200617

custom-header 1.5.8  installed via HACS

Config in raw editor:
```
custom_header:
  exceptions:
    - conditions:
        query_string: embed
      config:
        disable_sidebar: true
        kiosk_mode: true
```

Error (using chrome on mac):
```
Logger: frontend.js.latest.202006130
Source: components/system_log/__init__.py:209
First occurred: June 22, 2020, 3:32:36 PM (32 occurrences)
Last logged: 11:56:25 AM

http://raspberrypi2.lan:8123/frontend_latest/chunk.402d6d994eeade1f37bb.js:2:61210 Uncaught YAMLException: unacceptable kind of an object to dump [object Undefined]
http://raspberrypi2.lan:8123/hacsfiles/custom-header/custom-header.js:1:17373 Uncaught TypeError: Cannot read property 'cloneNode' of null
https://ha.derdik.com/hacsfiles/custom-header/custom-header.js:1:17373 Uncaught TypeError: Cannot read property 'cloneNode' of null
http://raspberrypi2.lan:8123/config/lovelace/resources:0:0 Uncaught
https://ha.derdik.com/hacsfiles/custom-header/custom-header.js?cb=1:1:17373 Uncaught TypeError: Cannot read property 'cloneNode' of null
```

**Comment by maykar** [comment]
Looks like some elements may have changed in the next round of updates. Will check this out when the beta is actually released.

**Comment by iderdik** [comment]
Ok, I've reverted to 0.111 for now and all is well. Thanks!

**Comment by maykar** [comment]
Latest release (1.6.0) adds support for 0.112 if you wanted to give it a test run.

**Comment by sysoppl** [comment]
I'm using Home Assistant 0.112.0.dev20200624 with custom-header 1.6.1 installed via HASS and I've got no text:
![obraz](https://user-images.githubusercontent.com/30871217/85630818-a2ae1280-b674-11ea-9cb1-e6825de7a23a.png)

**Comment by maykar** [comment]
Need to update the editors elements, thanks. Will have that in next release.

**Comment by maykar** [comment]
Fixed editor text in 1.6.2. Going to close this one. If any other issues in HA 0.112 are found, please create a new issue.
